bark/benchmark/tests/py_benchmark_process_tests.py

```diff
@@ -22,12 +28,14 @@
 from bark.runtime.commons.parameters import ParameterServer
 from bark.core.models.behavior import BehaviorIDMClassic, BehaviorConstantVelocity
 
-# to find database files
-os.chdir("../benchmark_database/")
+
 
 class DatabaseRunnerTests(unittest.TestCase):
     def test_database_run_and_analyze(self):
-        dbs = DatabaseSerializer(test_scenarios=4, test_world_steps=2, num_serialize_scenarios=2)
+        dbs = DatabaseSerializer(test_scenarios=2, test_world_steps=3, num_serialize_scenarios=2)
+        # to find database files
+        cwd = os.getcwd()
+        os.chdir("../benchmark_database/")
```

Comment:
why is the os.chdir necessary now? It does not seem to provide the same usability as before